amulety/utils.py

```diff
@@ -123,6 +129,8 @@ def process_airr(inpath: str, chain: str, sequence_col: str = "sequence_vdj_aa",
     data = pd.read_table(inpath)
     if "locus" not in data.columns:
         data.loc[:, "locus"] = data.loc[:, "v_call"].apply(lambda x: x[:3])
+
```

Comment:
Add code here to check receptor type is correct:
- For "BCR": check that the locus column contains IGH and IGL/K only. If it contains TCR chains as well then throw a warning that the TCR chains are being removed and only BCR used. If it doesn't contain any BCR chains, then throw an error that this embedding is trained only on TCR data and should not be used for BCR.
- For "TCR" same but opposite.
- For "All" it will generate embeddings for both BCR and TCR sequences in the file.